packages/mask/src/plugins/Profile/SNSAdaptor/PageTag.tsx

```diff
@@ -16,8 +16,8 @@ const useStyles = makeStyles()((theme) => ({
         color: theme.palette.mode === 'dark' ? 'rgba(239, 243, 244, 1)' : 'rgba(17, 20, 24, 1) ',
     },
     selected: {
-        border: `1px solid ${theme.palette.primary.main}`,
-        color: theme.palette.primary.main,
+        border: '1px solid rgb(29,155,240) !important',
```

Comment:
Please revert this. It should follow the twitter theme.

![image](https://user-images.githubusercontent.com/52657989/143765506-71e43471-0a78-4f71-b4a2-c3d28ad34c57.png)